composer.json

```diff
@@ -21,6 +22,6 @@
     }
   },
   "scripts": {
-    "test": "phpunit -c test"
+    "test": "php vendor/bin/phpunit -c test/phpunit.xml"
```

Comment:
This line should be functionally equivalent to the previous one. How does this fix code coverage?

Reply:
phpunit command option ```-c``` means ```--configuration```.
And I just specify the current path of ```phpunit.xml```.

Reply:
PHPUnit looks in the specified directory for `phpunit.xml`, so it should do the same thing. This pattern is used in other projects and works fine: https://github.com/ScriptFUSION/Porter/blob/master/composer.json#L39. For some reason, (probably because the tests are PHPT format,) it just doesn't work in this project. I cannot deny your change seems to fix the issue, but I suspect there's still a bug in PHPUnit itself.

If specifying the config file explicitly fixes the issue, does that mean the `php vendor/bin/` change isn't needed?